src/main/resources/assets/botania/lang/zh_CN.lang

```diff
@@ -478,13 +523,13 @@ achievement.botania:enchanterMake=Odds and Ends
 achievement.botania:enchanterMake.desc=制作一个魔力附魔台来更好的控制附魔
 achievement.botania:pollidisiacPickup=Romeo and Cinderella
 achievement.botania:pollidisiacPickup.desc=做一个畜牧花，在空气中制造爱情立场
-achievement.botania:brewPickup=Childhood Blues
+achievement.botania:brewPickup=オサナナブルー
 achievement.botania:brewPickup.desc=制作一个植物酿造台并用他酿造所有种类的药水
-achievement.botania:terraformRodCraft=World is Mine
+achievement.botania:terraformRodCraft=ワールドイズマイン
 achievement.botania:terraformRodCraft.desc=制作一把泰拉高并用他改变世界地形
 achievement.botania:bubbellPickup=深海少女
 achievement.botania:bubbellPickup.desc=制作一株气泡铃来方便水下建筑
-achievement.botania:luminizerRide=World's End Dancehall
+achievement.botania:luminizerRide=ワールズエンド ダンスホール
 achievement.botania:luminizerRide.desc=制作一些微光并用它来进行传输
```

Comment:
Good question.
Months ago, one of my friend realized that those achievements name are actually all vocaloid song names. Vazkii also mentioned this point in his new AMA thread but I am too lazy to find the link. 
Thus, thing becomes somehow tricky here: some of those songs have common Chinese name indeed (AFAIK, cause I am not fan of vocaloid), while rest of them doesn't. Using  hiragana can only tell user that they are ACG-related, but cannot tell they are vocaloid-related unless the user is also of vocaloid.
Also, I am trying applying a [guideline](https://github.com/Meow-J/Mod-Translation-Styleguide) on it right now, but this guideline only recommend to rewrite romanji to katakana/hiragana, and does not mention the solution of this case. 
I think I need to summon some folks. @Mrkwtkr @Meow-J